analytics/upress_reporting/models/data/download_event.py

```diff
@@ -0,0 +1,5 @@
+class DownloadEvent():
+    def __init__(self, title, timestamp, edition_id):
+        self.title = title
+        self.timestamp = timestamp
+        self.edition_id = edition_id
```

Comment:
nit: let's a add a new line! also feel free to use a data class for this https://docs.python.org/3/library/dataclasses.html